### ISSUE TITLE
Cant set Amaze as the default file manager on Android TV

### ISSUE THREAD

**Issue by alexanderadam** [TeamAmaze/AmazeFileManager#4079]
**Is your feature request related to a problem? Please describe.**

On Android TV (i.e. Chromecast) is no default file manager installed. It is possible to install Amaze but other applications still claim that there's no file manager installed.
However, with other file managers like [MaterialFiles](https://github.com/zhanghai/MaterialFiles/) this works.
Once they're installed other apps see this as an available file manager.

**Describe the solution you'd like**

It should be possible to set Amaze as a default file manager.

**Describe alternatives you've considered**

Using other file managers I guess? :thinking:

**Comment by muhamadsyafii** [comment]
Hi, I have access to an Android TV / Android Box device and would like to investigate this issue. I will first try to reproduce the behavior, compare Amaze's intent filters with another file manager that is detected correctly, and then propose a minimal fix if the problem is in the manifest. Can I work on it?

**Comment by VishnuSanal** [comment]
@sure - please do, thanks! :))

**Comment by muhamadsyafii** [comment]
Hi, I have opened PR #4707 for this issue:

https://github.com/TeamAmaze/AmazeFileManager/pull/4707

The change advertises Amaze as an `ACTION_OPEN_DOCUMENT` handler and
routes the request through the existing file-picking flow.

I verified it on two physical devices:

- Polytron Android TV, Android 7.1.2 (API 25)
- Amlogic Android Box, Android 9 (API 28)

Amaze now appears alongside Material Files in the system resolver.
Single and multiple file selection work, and selecting `ALWAYS` makes
subsequent matching requests open Amaze directly.

The relevant automated tests, debug builds, and `spotlessCheck` pass
locally.

A detailed technical write-up is available here:

https://pilkupil.my.id/article/making-amaze-discoverable-as-a-file-manager-on-android-tv

Thank you!
